src/elements/content-sidebar/__tests__/SidebarClassification-test.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Collapsible from '../../../components/collapsible';
+import SidebarClassification from '../SidebarClassification';
+
+describe('elements/content-sidebar/SidebarClassification', () => {
+    const getWrapper = props => shallow(<SidebarClassification {...props} />);
+
+    describe('render()', () => {
+        test('should render nothing when unclassified and can_upload is false', () => {
```

Comment:
Unclassified is actually an incorrect term, because unclassified is a sort of classification, you can have an unclassified label. Changing it to not classified.